### DISCUSSION TITLE
Enable navigation links to appear/hide on static Pages

### DISCUSSION THREAD

**Discussion by ellativity** [forem/forem#15974] [Feature Requests]
We would like to give Admins the option of hiding or displaying the navigation links on all their Forem's pages, including static Pages (e.g. https://dev.to/terms). 

Currently, the only way to access the navigation links from these pages is to return to the home screen (by clicking on the Forem name/logo in the top left corner) or using the back button to return to a page where navigation is displayed.

**Comment by amywtlin** [comment]
We do have plans for this within the Navigations project - see [Collapsible sections](https://docs.google.com/document/d/1tidFU5mR6vdTV1PMdAEUvLFKZHHqftwKTxo6YrVJHpc/edit#heading=h.ox01u0jnd6qf). [It](https://github.com/forem/rfcs/issues/302) is being queued up for technical assessment.


@ellativity Also, is this a piece of feedback from Creators? If so, I would like to capture that, though, I need to figure out a way of doing it. I am thinking about a tag for it but I'll bring this back to the team for discussion and see how we can capture it.

**Reply by robertandrews** [reply]
Hi, is there anything shareable about that "Navigations project"? One of the links is a private doc, the other is 404. Interested in what you're thinking.

**Reply by ellativity** [reply]
Hey @robertandrews these are some early stage design documents, so they're not really at a shareable stage.[ I'm linking to your feature request in Discussions here, so that our team can connect the dots](https://github.com/forem/forem/discussions/16959) - and hoping to be able to pass the baton to someone else to speak on this design decision over in that thread.
